digits/model/images/job.py

```diff
@@ -17,3 +21,16 @@ def __init__(self, **kwargs):
         super(ImageModelJob, self).__init__(**kwargs)
         self.pickver_job_model_image = PICKLE_VERSION
 
+    @override
+    def get_job_stats_as_json_string(self,epoch=-1):
```

Comment:
Sorry missed this comment! Yes that is a much better idea, I changed it in my last commit.